src/style.css

```diff
@@ -1,3 +1,147 @@
-body {
+html body {
+  font-family: Roboto, Arial, sans-serif;
+  --header-menu-active-color: rgba(0, 172, 220, 1);
+  --menu-apple-width: 37px;
+  --blue-underline-height: 4px;
+  --blue-underline-redius: 30%;
+  --total-length-horizontal-margins-inside-header: 140px;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  background: #fff;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+}
+
+.logo {
+  position: relative;
+  left: 50px;
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__link {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo__link img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+.nav {
+  position: relative;
+  right: 50px;
+}
+
+.nav__list {
+  font-size: 12px;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  width: calc(
```

Comment:
You dont need to calculate width here. Navigation will take as much width as it need based on the content